src/components/InfoBanner.tsx

```diff
@@ -1,18 +1,35 @@
+import ErrorIcon from '@mui/icons-material/Error';
 import InfoIcon from '@mui/icons-material/Info';
+import WarningIcon from '@mui/icons-material/Warning';
 import { Paper, PaperProps, Typography } from '@mui/material';
 
 interface InfoBannerProps extends PaperProps {
   text: string;
+  size?: 'small' | 'large';
+  type: 'info' | 'warning' | 'error';
 }
 
-export const InfoBanner = ({ text, sx, ...props }: InfoBannerProps) => {
+export const InfoBanner = ({ text, type, sx, size = 'large', ...props }: InfoBannerProps) => {
   return (
     <Paper
       elevation={5}
-      sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center', p: '1rem', bgcolor: 'info.main', ...sx }}
+      sx={{
+        display: 'flex',
+        gap: '0.5rem',
+        alignItems: 'center',
+        p: size === 'large' ? '1rem' : '0.3rem 2rem',
```

Comment:
Hvordan funker det at det er to størrelser når size ikke er large?

Reply:
Det er `paddingX` og `paddingY` :+1: